exercises/say/README.md

```diff
@@ -9,17 +9,17 @@ Handle the basic case of 0 through 99.
 If the input to the program is `22`, then the output should be
 `'twenty-two'`.
 
-Your program should complain loudly if given a number outside the
-blessed range.
+Your program should throw an error if given a number outside the
+range of 0 through 99.
 
 Some good test cases for this program are:
 
 - 0
 - 14
 - 50
 - 98
-- -1
-- 100
+- 99 
+- 100 (currently an error case)
```

Comment:
With changing the function to use an unsigned `int64` I don't think we need to stray from the description in the `problem-specifications` repo.